src/navigation/navigation.hpp

```diff
@@ -161,6 +166,8 @@ namespace navigation {
       // number of previous measurements stored
       static constexpr int kPreviousMeasurements = 1000;
 
+      static constexpr char delimiter = '\t';
```

Comment:
Seriously? :laughing: You didn't even need to look at the [constant naming section](https://hyp-ed.github.io/styleguide/#Constant_Names) of the style guide to get this right - all the lines around have the correct format.
```suggestion
      static constexpr char kDelimiter = '\t';
```

Reply:
Yeah honest mistake, thought the k was only used for numbers, oops lol